### DISCUSSION TITLE
Axis camera stops responding to ONVIF requests and has to be restarted

### DISCUSSION THREAD

**Discussion by xtrdustin** [AxisCommunications/discussions#114] [General] [answered]
Our Axis camera, on a couple occasions, has stopped responding to ONVIF requests and had to be restarted. We've seen this twice that I know of (about a week ago and this morning). Any ONVIF requests (even the unauthenticated "get system time" one) got a socket timeout exception, but video feeds seemed to work. 

The camera's web interface worked except for the System -> ONVIF page. That page wouldn't load ONVIF users or media profiles. The column headers were there, but they never populated, and there was a loading spinner that never finished below the "media profiles" column.

Camera: AXIS Q6215-LE PTZ Network Camera

Current firmware: 9.80.3.14

I've uploaded a crash report to the private github repo, which @mattias-kindborg-at-work  has access to: https://github.com/xtrdustin/axis-uploads/tree/main/CrashReports

**Comment by mattias-kindborg-at-work** [comment, marked as answer]
Hi @xtrdustin and thanks for your bug report! To enable you to get the best help possible when it comes to product specific bugs, like this one, you will get much better help at https://www.axis.com/support/helpdesk. The Helpdesk is much better tailored to make changes to AXIS OS, which seems to be the problem according to your description of the issue, and they will probably also benefit from your contact if they are having problems with reproducing the issue.

Again, thanks for notifying us!

**Reply by xtrdustin** [reply]
Hi Mattias, thanks for getting back to me. Created help desk ticket 02033550 for this issue.
